### DISCUSSION TITLE
Wiki Usage

### DISCUSSION THREAD

**Discussion by CarterBeauBenson** [CommonCoreOntology/CommonCoreOntologies#554] [Ontology Terms and Class Schema]
It looks like the Wiki has mostly been untouched since 2021. This discussion is meant to kick off a conversation on how to update the wiki.

If I am interpreting it correctly, it looks like the wiki page was historically used in a way that "discussions" is intended to be used now. 

My suggestion is to either scrub it and move some of the discussions that are still pertinent to "discussions" or to move it all under one tab called "legacy Wiki" or something. I suggest we then add updated whitepapers. This includes "Getting Started with CCO," "An Overview of the Common Core Ontologies," "Best Practices," and "Modelling Information."

**Comment by neilotte** [comment]
@mark-jensen @CarterBeauBenson @johnbeve @APCox this strikes me as a good winter break task.

**Comment by CarterBeauBenson** [comment]
Just want to ping this

**Comment by johnbeve** [comment]
I'm in favor of using the CCO wiki as a source for documentation of the sort Carter describes. GitHub wikis are indeed used for [that](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis), expanding beyond readmes. 

Perhaps this could be on the model of the [OBO Textbook](https://oboacademy.github.io/obook/)

Additionally, some examples of a few [well-done wikis]( https://project-awesome.org/MyHoneyBadger/awesome-github-wiki#examples) for inspiration.

**Comment by CarterBeauBenson** [comment]
For example, having tabs that run down the side?
1. Getting started with CCO
2. CCO design patterns
3. Modeling information using CCO
4. ....

I think OBO might have too much stuff. It is overwhelming
